src/BaseTokenWrapper.sol

```diff
@@ -103,6 +103,14 @@ abstract contract BaseTokenWrapper is Ownable, IBaseTokenWrapper {
     return _withdrawToken(amount, to, aTokenOut);
   }
 
+  /// @inheritdoc IBaseTokenWrapper
+  function borrowToken(uint256 amount, address to) external virtual {
+    require(amount > 0, 'INSUFFICIENT_AMOUNT_TO_BORROW');
+    POOL.borrow(TOKEN_OUT, amount, 2, 0, address(to));
+    uint256 amountIn = _unwrapTokenOut(amount);
```

Comment:
let's do similar to `withdraw`and use the real balance it holds (compute difference of balance)

Reply:
updated in 340414fd777c2012469137465a4a20aab48fd5ef , just double check this one is correct